FreshRSS/FreshRSS
# Add: <meta name="theme-color">

Closes #5088

Changes proposed in this pull request:

- via theme's `metadata.json` is it now possible to give a `theme-color`
- each theme has now the color defined in its `metadata.json`

metadata.json
```
{
	"name": "Dark",
	.....
	"theme-color": "#111111"
}
```
`<meta name="theme-color" content="#111111">`
(`"theme-color": {"default": "#111111"}` would do the same)

Origine theme has a CSS color scheme that respects the operating system/browser preference for light/dark themes
```
{
	"name": "Origine",
	.....
	"theme-color": {"dark": "#1f1f1f", "light": "#f0f0f0"}
}
```
`<meta name="theme-color" media="(prefers-color-scheme: light)" content="#f0f0f0">`
`<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1f1f1f">`


How to test the feature manually:
![grafik](https://user-images.githubusercontent.com/1645099/218539505-157bd2b2-702b-4796-b150-f9fe3b36db09.png)
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color

1. use Safari Browser on iPhone (Chrome on Android does not respect the `dark`, but works fine with `light`
2. see the color of address bar and operating system bar


Pull request checklist:
